addons/event/models/event_ticket.py

```diff
@@ -200,15 +200,3 @@ def _unlink_except_if_registrations(self):
             raise UserError(_(
                 "The following tickets cannot be deleted while they have one or more registrations linked to them:\n- %s",
                 '\n- '.join(self.mapped('name'))))
-
-    def _get_ticket_printing_color(self):
-        self.ensure_one()
-        default_color = '#000000'
-        color_overrides_json = self.env['ir.config_parameter'].sudo().get_param('event.ticket_text_colors')
-        if color_overrides_json:
-            try:
-                color_overrides = json.loads(color_overrides_json)
```

Comment:
Is the 'json' import still used? I see the PR introducing this code added the import, but this PR is not removing it, maybe it's used somewhere else now.